CRM/Utils/Request.php

```diff
@@ -137,6 +137,34 @@ public static function retrieve($name, $type, &$store = NULL, $abort = FALSE, $d
   }
 
   /**
+   * @param string $name
+   *   Name of the variable to be retrieved.
+   *
+   * @param array $method - '$_GET', '$_POST' or '$_REQUEST'.
+   *
+   * @return mixed
+   *    The value of the variable
+   */
+  public static function getValue($name, $method) {
+    if (isset($method[$name]) {
```

Comment:
Aside: Test infra wasn't running `civilint` correctly, which is why the problem showed up oddly. Thanks to @seamuslee001 for pushing this question.

This problem relates to yesterday's change in test infra -- i.e. it now applies patches using (essentially) `curl https://github.com/.../pull/1234.patch | git am -` . But `git am` produces a different a `sha1` for the final commit, and `civilint` was checking for a specific sha1.

Should be fixed now. Regenerated test output so that the last commit shows normal `civilint` output normally.